lib/cartopy/mpl/feature_artist.py

```diff
@@ -96,6 +96,12 @@ def __init__(self, feature, **kwargs):
             kwargs = {}
         self._kwargs = dict(kwargs)
 
+        if 'color' in self._kwargs:
+            # We want the user to be able to override both face and edge
+            # colours if the original feature already supplied it.
+            color = self._kwargs.pop('color')
+            self._kwargs['facecolor'] = self._kwargs['edgecolor'] = color
```

Comment:
I think these should be using ``setdefault``, not setitem...

```
Feature(..., edgecolor='red', color='black')._kwargs == {'edgecolor': 'black', 'facecolor': 'black'}
```

What I'd expect with the above call is ```{'edgecolor': 'red', 'facecolor': 'black'}```

Reply:
I was 50/50 on which one should override the other. I couldn't figure out which one has precedence in Matplotlib from a quick glance.

Reply:
@WeatherGod pointed out that there is a precedence in Matplotlib which is applied [here](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/artist.py#L986) and gives `color` precedence over the others, so this should be working in the correct way.